app/editor/src/features/admin/filters/utils/generateQueryForAction.ts

```diff
@@ -0,0 +1,38 @@
+export const generateQueryForAction = (
```

Comment:
I tried to make this as generic and reusable as possible, but it's still essentially a two path solution.  It supports a single wildcard search, or a specific value search.